model/patch_lifecycle.go

```diff
@@ -143,28 +143,33 @@ func GetPatchedProject(ctx context.Context, p *patch.Patch, githubOauthToken str
 		hash = p.GithubPatchData.MergeCommitSHA
 	}
 
+	var path = projectRef.RemotePath
```

Comment:
nit: we typically define new variables using := instead of "var" to be more clear

Reply:
Fixed in latest commit